components/BUILD.yaml

```diff
@@ -15,19 +15,6 @@ packages:
       - install/installer:docker
       - install/kots:lint
       - components/gitpod-protocol:all
-
```

Comment:
We don't need this target anymore and we don't need the artefacts copied to /tmp anymore.